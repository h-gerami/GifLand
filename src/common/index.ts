export * from './image/LazyLoadImage';
export * from './searchbar/SearchBar';
export * from './loading/Loading';
export * from './button/ReloadButton';
export * from './button/CopyUrlButton';
