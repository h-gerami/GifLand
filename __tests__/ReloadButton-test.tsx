import React from 'react';
import renderer from 'react-test-renderer';
import {ReloadButton} from '../src/common';

test('renders correctly', () => {
  const tree = renderer
    .create(<ReloadButton onReloadButtonClickHandler={() => console.warn(2)} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
