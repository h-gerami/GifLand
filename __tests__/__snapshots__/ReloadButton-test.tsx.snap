// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#fff",
        "borderRadius": 15,
        "elevation": 5,
        "height": 100,
        "justifyContent": "center",
        "opacity": 1,
        "shadowColor": "#000",
        "shadowOffset": Object {
          "height": 2,
          "width": 0,
        },
        "shadowOpacity": 0.25,
        "shadowRadius": 3.84,
        "width": 100,
      }
    }
  >
    <Text
      allowFontScaling={false}
      selectable={false}
      style={
        Array [
          Object {
            "color": "#2A3756",
            "fontSize": 50,
          },
          undefined,
          Object {
            "fontFamily": "Material Design Icons",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {},
        ]
      }
    >
      ó°‘“
    </Text>
    <Text
      style={
        Object {
          "color": "#2A3756",
          "fontFamily": "SofiaProBold",
          "fontSize": 15,
        }
      }
    >
      Reload
    </Text>
  </View>
</View>
`;
